recipe.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "your food's name",
+  "ingredients": [
+    "ingredient1",
+    "ingredient2",
+    "ingredient3"
+  ],
+  "instructions": [
+    "your instruction goes here"
+  ],
+  "total calories": "your number of total calories goes here",
```

Comment:
While this is technically a valid key, a common practice is to not use spaces in key names in JSON. Think of it as of variable name - this is what you're gonna be using in your code. `total_calories` is probably better

Reply:
You're right!